specs/git.spec.js

```diff
@@ -133,7 +134,7 @@ describe( "git", () => {
 				expectedRunCommandArgs: { args: "tag --sort=v:refname", logMessage: "Getting list of tags" }
 			},
 			shortLog: {
-				expectedRunCommandArgs: { args: "--no-pager log --no-merges --date-order --pretty=format:'%s'", logMessage: "Parsing git log" }
+				expectedRunCommandArgs: { args: "--no-pager log --no-merges --date-order --pretty=format:\"%s\"", logMessage: "Parsing git log" }
```

Comment:
Could you please update this to be surrounded by `` and then use non-escaped double quotes inside? I feel that it reads cleaner than escaped characters. Thanks